src/processor/operator/persistent/reader/csv/driver.cpp

```diff
@@ -35,9 +35,13 @@ void ParsingDriver::addValue(
             stringFormat("Error in file {}, on line {}: expected {} values per row, but got more.",
                 reader->fileInfo->path, reader->getLineNumber(), reader->numColumns));
     }
-
-    function::CastString::copyStringToVector(
-        chunk.getValueVector(columnIdx).get(), rowNum, value, &reader->option);
+    try {
```

Comment:
Can u try to load comment.csv again after this PR.

I always heard that try catch has a performance issue even if the catch block is never triggered.

If the performance stays the same. The rest looks good to me.